app/views/draft_mailer/draft_published_notification.html.erb

```diff
@@ -10,7 +10,7 @@
     <%= @user[:name] %>,
   </p>
   <p>
-    <%= "Your collection metadata record #{@short_name}_#{@version} has been successfully #{ @is_update ? 'updated' : 'published to the CMR'}. Your collection's concept ID is #{@concept_id}." %>
+    <%= "Your collection metadata record #{@short_name}_#{@version} has been successfully #{ @is_update ? 'updated' : "published to the CMR#{@email_env_note}"}. Your collection's concept ID is #{@concept_id}." %>
```

Comment:
<%= "Your collection metadata record #{@short_name}_#{@version} has been successfully #{ @is_update ? 'updated in' : 'published to'} the CMR#{@email_env_note}. Your collection's concept ID is #{@concept_id}." %>